### DISCUSSION TITLE
Problems on adding  characters

### DISCUSSION THREAD

**Discussion by juliuwu** [rayanmargham/FNF-Godot-Engine#17] [Q&A] [answered]
I follow the docks to add characters with png and xml, I converted the things on .res and add it on the animated sprite, but nothing happens, the sprites don't appears , i don't know if I miss something cause I try this 2 times

![Captura de tela 2021-12-08 221140](https://user-images.githubusercontent.com/77576393/145316237-67cbb17e-b9fe-4b86-852f-336addf86377.png)

**Comment by juliuwu** [comment]
I open the godot cmd and this appears to me when I try to use the XML convertor
![Captura de tela 2021-12-09 132928](https://user-images.githubusercontent.com/77576393/145436407-2e80f6d7-43e9-474f-b36b-86e50f0b325b.png)

**Reply by juliuwu** [reply]
Omg I'm so dumb

**Reply by rayanmargham** [reply]
nah its ok we all make mistakes

**Reply by rayanmargham** [reply]
i suggest you take a look at the code for a better understanding by the way
it'll help you in the future

**Reply by juliuwu** [reply]
thanks fella, I will study the code

**Comment by rayanmargham** [comment, marked as answer]
You inputed an incorrect format
do res://Assets/Stages/Characters/YOUR CHARACTER
